pkgs/servers/conduit/default.nix

```
@@ -0,0 +1,21 @@
+{ stdenv, lib, fetchFromGitLab, rustPlatform, }:
+rustPlatform.buildRustPackage rec {
+  pname = "conduit";
+  version = "git";
```

```suggestion
  version = "unstable-2021-MM-DD";
```

We usually reference the date in this format for unstable versions.